src/Files.Core/Helpers/NativeFindStorageItemHelper.cs

```diff
@@ -76,7 +76,7 @@ public struct WIN32_FIND_DATA
 			public string cAlternateFileName;
 		}
 
-		[DllImport("api-ms-win-core-file-fromapp-l1-1-0.dll", SetLastError = true, CharSet = CharSet.Unicode)]
```

Comment:
Need help with someone with Win32 knowledge to know if this `CharSet `property is important, the only equivalent is [`StringMarshalling`](https://learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.stringmarshalling?view=net-8.0)